sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusConnectionStringProperties.java

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.core.amqp.implementation.ConnectionStringProperties;
+import com.azure.core.credential.TokenCredential;
+
+import java.net.URI;
+
+/**
+ * A utility class that parses a connection string into sections.
+ * A Service Bus connection string is a set of key value pairs separated by semi-colon. A typical example is
+ * "Endpoint=sb://sbnamespace.servicebus.windows.net/;SharedAccessKeyName=someKeyName;SharedAccessKey=someKeyValue".
```

Comment:
Maybe adding a link to this article will be good:
https://docs.microsoft.com/azure/service-bus-messaging/service-bus-quickstart-portal#get-the-connection-string